cmd/medius/common/registry.go

```diff
@@ -100,7 +104,7 @@ func gatherArtifacts(registry *[]Entry, gatherers []api.ArtifactsGatherer) {
 		} else {
 			for i := range artifacts {
 				*registry = append(*registry, Entry{
-					Artifact:     artifacts[i],
```

Comment:
I think you reviewed old revision. This is how it looks like today:
```
func gatherArtifacts(registry *[]Entry, gatherers []api.ArtifactsGatherer) {
	for _, gatherer := range gatherers {
		artifacts, err := gatherer.Gather()
		if err != nil {
			logrus.Warn("Failed to gather artifacts", err)
		} else {
			for i := range artifacts {
				*registry = append(*registry, Entry{
					Artifacts:    []api.Artifact{artifacts[0][i]},
					UseForDocs:   i == 0,
					UseForLatest: i == 0,
				})
			}
		}
	}
}
```

Reply:
I reviewed the "left" side. The new is still wrong, since like this it probably run out of range. `i` is a range over the outer slice and you use it to access the inner slice. The `artifacts` returned by the gatherer should be [][]api.Artifact, then you can set each outer slice item as []api.Artifact in the loop below with `artifacts[i]`. IMO the gatherer should construct the multi-arch Artifact already. Like this you end up with a separate artifact for each architecture and version.

Reply:
Ah got it, thanks Felix! :)